### PR TITLE
l10n: Czech Translation

### PR DESCRIPTION
**Pull request by xskipy**

### CODE REVIEW

File: po/LINGUAS [hunk 1]
```diff
@@ -1,6 +1,7 @@
 be
 be@latin
 bg
+cs_CZ
```
Comment (martinpitt) [inline po/LINGUAS]: Please name this just "cs" -- we should avoid country specific variants in most cases, except for a few known cases (en, es, and zh).

File: po/cs_CZ.po [hunk 2]
```diff
@@ -0,0 +1,588 @@
+# translation of pl.po to Polish
+# Piotr Drąg <piotrdrag@gmail.com>, 2011, 2013, 2014, 2015, 2016.
+# Zbigniew Jędrzejewski-Szmek <zbyszek@in.waw.pl>, 2011.
+#
```
Comment (martinpitt) [inline po/cs_CZ.po]: Copy-paste errors; there is more "Polish" below.

File: po/LINGUAS (hunk 1 above)
Comment (martinpitt) [reply po/LINGUAS]: You still need to change the locale here too, not just rename the file.

**Comment by xskipy** [comment]
I'm new to git so it takes me a while to get it done correctly, if you could squash the commits into one for me I'd be really grateful

File: po/LINGUAS (hunk 1 above)
Comment (lnykryn) [inline po/LINGUAS]: cs_CZ is a known exception. cs in the past meant Czechoslovakia. And then we have the czech (CZ) part.

**Comment by xskipy** [comment]
I've reworked it, squashed it into one commit, but left the cs_CZ tag since it's a known Czech language tag.

**Comment by lnykryn** [comment]
cs_CZ is a known exception. cs in the past meant Czechoslovakia. And then we have the czech (CZ) part.

**Comment by martinpitt** [comment]
> cs_CZ is a known exception. cs in the past meant Czechoslovakia. 

Not known to me -- and "cs" did and does _not_ mean "Czechoslovakia" or any other country -- it's the name (or ISO code) for a _language_ (i. e. "Czech"), not a country. English, Spanish, Portugese, and Chinese are special because these have major different languages in different countries where doing by-country translations actually is required; but that is not the case for any other language I'm aware of.

**Comment by xskipy** [comment]
> Not known to me -- and "cs" did and does not mean "Czechoslovakia" or any other country -- it's the name (or ISO code) for a language (i. e. "Czech"), not a country. English, Spanish, Portugese, and Chinese are special because these have major different languages in different countries where doing by-country translations actually is required; but that is not the case for any other language I'm aware of.

[localeplanet.com/icu/cs-CZ](http://www.localeplanet.com/icu/cs-CZ/index.html) If you look it up, you can see it is widely used this way

**Comment by martinpitt** [comment]
You are mixing up a locale definition (which always includes a country) with a translation into a particular language (which should never include a country except in the mentioned four special cases).

**Comment by xskipy** [comment]
Very well, you convinced me, It's fixed. Thanks for clarification

**Comment by martinpitt** [comment]
Thanks!

**Comment by AsciiWolf** [comment]
I think this translation still needs some polishing as it unfortunately contains a lot of typos and other errors (incorrect spacing, bad escaping etc.). Anyway, great job! I planned to do the Czech translation myself some time ago, but did not find time for it, so I'm really glad someone did it. Thanks!

**Comment by keszybz** [comment]
@AsciiWolf: pull requests are welcome. I don't think there's that many users of localization, so if you see any typos please fix them. Even through the github web interface, for this kind of patches it's appropriate.

**Comment by AsciiWolf** [comment]
@keszybz: I have made a new Pull Request (https://github.com/systemd/systemd/pull/4203).

**Pull request merged**